t/cli/test_main.sh

```diff
@@ -164,8 +164,8 @@ if [ $count_https_specific_ip -ne 2 ]; then
     exit 1
 fi
 
-count_https_specific_ip_and_enable_http2=`grep -c "listen 127.0.0..:944. ssl default_server http2" conf/nginx.conf || true`
-if [ $count_https_specific_ip_and_enable_http2 -ne 1 ]; then
+count_https_specific_ip_and_enable_http2=`grep -c "listen 127.0.0..:944. ssl default_server" conf/nginx.conf || true`
+if [ $count_https_specific_ip_and_enable_http2 -ne 2 ]; then
```

Comment:
I think you should check the `http2` directive?

Reply:
OK, I will do it.

Reply:
PTAL, it should be fixed by https://github.com/apache/apisix/pull/10887/commits/22d46db26f75ec385fec203b204896ba4824c4ed